docs/howto/openstack/nova/boot-image-volume.md

```diff
@@ -0,0 +1,395 @@
+# Converting boot-from-image servers to boot-from-volume
+
+You can freely move any server [between {{brand}}
+regions](move-server-between-regions), provided it uses
+boot-from-volume. Since such servers offer much more flexibility than
+those using boot-from-image, we recommend the former over the latter.
+You may still have boot-from-image servers, though. If you decide to
```

Comment:
Here, we'll need to explain how the user can tell the difference between a boot-from-volume and a boot-from image server. One way to do it would be to check the server's volume attachments, and see if a volume is attached as `/dev/vda` (or `/dev/sda`), i.e. the server's boot volume. Another way is the approach you use in the _Viewing the new server_ section.

Reply:
Understood. I will expand on that.